pkg/ansi/ansi.go

```diff
@@ -36,6 +36,12 @@ var DisableColors = false
 // `CLICOLOR_FORCE`. Cf. https://bixense.com/clicolors/
 var EnvironmentOverrideColors = true
 
+// HostStdoutIsTerminal enables color on stdout output when this package is used in a plugin.
+var HostStdoutIsTerminal = true
+
+// HostStderrIsTerminal enables color on stderr output when this package is used in a plugin.
+var HostStderrIsTerminal = true
+
```

Comment:
These are global variables that the plugin process sets at runtime, after it receives the rpc request from the host and reads the `AdditionalInfo` struct.

Reply:
I don't see where the plugin sets that information, is that handled on a per plugin basis, iow is a plugin expected to call this interface if they care about the information?

Reply:
These variables will be set by the plugin bootstrap library that all plugins import: https://github.com/stripe-dev/stripe-cli-plugin-bootstrap/pull/3. Individual plugin code will not handle any of this.

Reply:
Ah, excellent, that's what I was looking for -- I had missed that repo